datasets/us/cuba_sanctions/crawler.py

```diff
@@ -51,11 +54,14 @@ def crawl_restricted_entities(context: Context):
     landing_doc = fetch_html(
         context, context.dataset.url, CONTENT_XPATH, actions=ACTIONS, cache_days=1
     )
+    # The restricted list URL page currently points to an interim page that refers to ORIGINAL_ACCOMMODATIONS_URL.
+    # So the page doesn't really contain that list. We just want to alert when it changes
     restricted_list_url = landing_doc.xpath(".//a[text()='Cuba Restricted List']/@href")
     assert len(restricted_list_url) == 1, restricted_list_url
-    doc = fetch_html(context, restricted_list_url[0], CONTENT_XPATH, actions=ACTIONS)
-    node = doc.find(CONTENT_XPATH)
-    if not h.assert_dom_hash(node, "52894f39893ab2432b411441f3bc21c27c3bea7d"):
+    restricted_list_unblock_validator = ".//div[@id='content']"
+    doc = fetch_html(context, restricted_list_url[0], restricted_list_unblock_validator)
+    node = doc.find(restricted_list_unblock_validator)
```

Comment:
just a naming nit, in this context it's just the xpath to the content section of the page. It just so happens that a sensible "unblock validator" is usually the first xpath you're going to use once the page is fetched.
```suggestion
    content_xpath = ".//div[@id='content']"
    doc = fetch_html(context, restricted_list_url[0], content_xpath)
    node = doc.find(content_xpath)
```